src/EnergyPlus/BoilerSteam.cc

```diff
@@ -456,28 +462,49 @@ namespace BoilerSteam {
                             "Boiler " + this->BoilerFuelTypeForOutputVariable + " Rate",
                             OutputProcessor::Unit::W,
                             this->FuelUsed,
-                            "System",
-                            "Average",
+                            OutputProcessor::eTimeStepType::System,
+                            OutputProcessor::eVariableType::Average,
                             this->Name);
         SetupOutputVariable(state,
                             "Boiler " + this->BoilerFuelTypeForOutputVariable + " Energy",
                             OutputProcessor::Unit::J,
                             this->FuelConsumed,
-                            "System",
-                            "Sum",
+                            OutputProcessor::eTimeStepType::System,
+                            OutputProcessor::eVariableType::Sum,
                             this->Name,
                             _,
                             this->BoilerFuelTypeForOutputVariable,
                             "Heating",
                             this->EndUseSubcategory,
                             "Plant");
-        SetupOutputVariable(state, "Boiler Steam Efficiency", OutputProcessor::Unit::None, this->BoilerEff, "System", "Average", this->Name);
-        SetupOutputVariable(
-            state, "Boiler Steam Inlet Temperature", OutputProcessor::Unit::C, this->BoilerInletTemp, "System", "Average", this->Name);
-        SetupOutputVariable(
-            state, "Boiler Steam Outlet Temperature", OutputProcessor::Unit::C, this->BoilerOutletTemp, "System", "Average", this->Name);
-        SetupOutputVariable(
-            state, "Boiler Steam Mass Flow Rate", OutputProcessor::Unit::kg_s, this->BoilerMassFlowRate, "System", "Average", this->Name);
+        SetupOutputVariable(state,
```

Comment:
Sometimes the new length caused Clang Format to break lines, but no functional change here, once again just the two arguments.